build_misc/moe.absolucy.logcleaner.plist

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Label</key>
+	<string>moe.absolucy.logcleaner</string>
+	<key>ProgramArguments</key>
+	<array>
+		<string>@MEMO_PREFIX@@MEMO_SUB_PREFIX@/bin/find</string>
```

Comment:
Does this not cause an issue? I thought you needed a wrapper to call on some bins. If it works this way, just leave it. Otherwise, add a wrapper and use that instead.